Source/BlueprintSearchVSExtension/Source/Commands/CommandHelpers/PathFinderHelper.cs

```diff
@@ -0,0 +1,89 @@
+//Copyright(C) Coconut Lizard Limited. All rights reserved.
+
+// ---------------------------------------------------------
+
+using EnvDTE;
+using EnvDTE80;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Windows;
+
+namespace BlueprintSearch.Commands.CommandHelpers
+{
+	public static class PathFinderHelper
+	{
+		public static DTE2 DTEService;
+
+		public static string UProjectFilePath { get; set; }
+
+		public static string UECommandLineFilePath { get; set; }
+
+		public static void FindPaths()
+		{
+			List<Project> ProjectsList = new List<Project>();
+			ProjectsList.AddRange(DTEService.Solution.Projects.Cast<Project>());
+
+			for (int i = 0; i < ProjectsList.Count; i++)
+			{
+				ProjectsList.AddRange(ProjectsList[i].ProjectItems.Cast<ProjectItem>().Select(x => x.SubProject).OfType<Project>());
+			}
+
+			for (int i = 0; i < ProjectsList.Count; ++i)
+			{
+
+				if (ProjectsList[i].Name.Equals("UE4"))
+				{
+					// This solution contains a ue4 project, find the path!
+					string EngineIncludePath = GetUnrealCommandLineExecutablePathFromProject(ProjectsList[i]);
+					UECommandLineFilePath = EngineIncludePath;
+					if (EngineIncludePath.Equals(string.Empty))
```

Comment:
In C# it looks like you're able to call instead:
```suggestion
					if (string.IsNullOrEmpty(EngineIncludePath))
```